### PR TITLE
Update information for collector compression

### PR DESCRIPTION
**Pull request by ajaiswals**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/data-source-configurations/index.md [hunk 1]
```diff
@@ -10,9 +10,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 Data sources or Source in Sumo refers to a specific application/infrastructure component from which the collector can collect telemetry (Logs, Metrics, and Traces). The sources include all configuration including protocol, receivers, processors, and exporters required to collect telemetry data from a given source. MySQL, Nginx, Kafka, Cassandra, and JMX are all examples of sources from where Sumo Logic collector can collect and send data. 
 
 
-### Configuration
+### Configuration 
 To collect data from a source the OpenTelemetry collector requires **source configuration**, that instructs the collector how to collect the data. [Learn More](https://opentelemetry.io/docs/collector/configuration/) about OpenTelemetry Collector configuration. 
 
+
+### How  OpenTelemetry Collector sends data to the Sumo service
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/index.md:17]:
```suggestion
### How the OpenTelemetry Collector sends data to Sumo Logic
```

File: docs/send-data/opentelemetry-collector/data-source-configurations/index.md [hunk 2]
```diff
@@ -10,9 +10,13 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+OpenTelemetry Collector starts sending data to the Sumo service as soon as it is available from the Sources configured on the Collector. Before sending the data, a Collector compresses (by a factor of 10x) and encrypts the data. A Collector sends data to the Sumo service over HTTPS using SumoLogic Exporter.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/data-source-configurations/index.md:18]:
```suggestion
Our OpenTelemetry Collector starts sending data to the Sumo Logic service as soon as it is available from the Sources configured on the Collector. Before sending the data, a Collector compresses (by a factor of 10x) and encrypts the data. A Collector sends data to the Sumo service over HTTPS using the SumoLogic Exporter.
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor suggestions - thanks for submitting, @ajaiswals !

**Comment by ajaiswals** [comment]
> minor suggestions - thanks for submitting, @ajaiswals !

Changes accepted, thanks for prompt review.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**